src/UIAutomation/UiaOperationAbstraction/UiaTypeAbstraction.g.h

```diff
@@ -2068,6 +2068,8 @@
         UiaCustomNavigationPattern GetCustomNavigationPattern(bool useCachedApi);
         UiaSelectionPattern2 GetSelectionPattern2(bool useCachedApi);
 
+        UiaElement BuildUpdatedCache(UiaCacheRequest cacheRequest);

```

Comment:
If you think that stylistically, the blanks/spaces make no sense, let me know. :)

Reply:
looks good